v2/alias.go

```diff
@@ -135,11 +135,14 @@ var (
 
 	// Context
 
-	ContextWithTarget               = context.WithTarget
-	TargetFromContext               = context.TargetFrom
-	ContextWithRetriesLinearBackoff = context.WithRetriesLinearBackoff
-	WithEncodingBinary              = binding.WithForceBinary
-	WithEncodingStructured          = binding.WithForceStructured
+	ContextWithTarget                    = context.WithTarget
+	TargetFromContext                    = context.TargetFrom
+	ContextWithRetriesConstantBackoff    = context.WithRetriesConstantBackoff
+	ContextWithRetriesLinearBackoff      = context.WithRetriesLinearBackoff
+	ContextWithRetriesExponentialBackoff = context.WithRetriesExponentialBackoff
```

Comment:
growing and growing :smile:

Reply:
it is better than all the source in a single dir

Reply:
yeah but before the v2 i wish to review it to check what we need and what not